source/kong_tcpingress.go

```diff
@@ -192,16 +191,6 @@ func (sc *kongTCPIngressSource) filterByAnnotations(tcpIngresses []*TCPIngress)
 	return filteredList, nil
 }
 
-func (sc *kongTCPIngressSource) setDualstackLabel(tcpIngress *TCPIngress, endpoints []*endpoint.Endpoint) {
```

Comment:
The test is removed, which is understandable, but what about a test where ipv4 with ipv6 ?

Reply:
Not a test, just redundant label functionality.